### ISSUE TITLE
Why does Tweak have params in init that aren't allowed to be used?

### ISSUE THREAD

**Issue by CraigSiemens** [mixpanel/mixpanel-swift#187]
Both `collectionName` and `groupName` are parameters on the init for `Tweak` but it seems like they should never be set, and setting them to any value causes the Tweak to no longer appear in the web A/B Testing interface.

```
public init(tweakName: String,
	            defaultValue: T,
	            min minimumValue: T? = nil,
	            max maximumValue: T? = nil,
	            stepSize: T? = nil,
	            _ collectionName: String = "Mixpanel",
	            _ groupName: String = "Mixpanel")
```

The docs say
```
- parameter collectionName: the collection name of the tweak (do not set, optional)
- parameter groupName:      the group name of the tweak (do not set, optional)
```

If they aren't useable, wouldn't it make more sense for them to not be part of the public init?

**Comment by zihejia** [comment]
We will have a look and get back to you

**Comment by zihejia** [comment]
Thanks @CraigSiemens for raising this. We have released 2.5.3 and it has a fix for this.
